### ISSUE TITLE
Hono: Data-grid pod doesn't get ready

### ISSUE THREAD

**Issue by calohmn** [eclipse/packages#61]
I've deployed the Hono chart in a [kind](https://kind.sigs.k8s.io/) cluster using the `dataGridExample.enabled=true` option to deploy an Infinispan data grid for the Hono device connection service.

The `eclipse-hono-data-grid-0` pod didn't get ready here, also causing the protocol adapter pods to not get ready.
Error in the `eclipse-hono-data-grid-0` pod:
`Readiness probe errored: rpc error: code = DeadlineExceeded desc = failed to exec in container: timeout 3s exceeded: context deadline exceeded`.

When running the readiness probe command ([`/usr/local/bin/is_healthy.sh`](https://github.com/jboss-dockerfiles/infinispan/blob/9.4.x/server/is_healthy.sh)) manually in the pod, the command took 22s - so way more than the 3s timeout defined in the chart.

I think the timeout should be increased here. Better yet would be to update to Infinispan >= 10, where there is a proper [readinessProbe REST endpoint](https://github.com/infinispan/infinispan-images#liveness-and-readiness-probes).

**Comment by kaniyan** [comment]
Also I executed the readiness probe [command ](https://github.com/jboss-dockerfiles/infinispan/blob/9.4.x/server/is_healthy.sh)manually in the pod deployed in the _minikube_ in my local machine, it look 26 seconds.

**Comment by sophokles73** [comment]
@calohmn @kaniyan When did you run the `is_healthy.sh` command? Did it take that long on first invovation only and once the grid was up the command took only a second or less? Or does it take that long for you on each invocation?

**Comment by calohmn** [comment]
@sophokles73 For each invocation, also once the grid is running. Just did another test on minikube: grid hast started, pod is ready and manually invoking `is_healthy.sh` takes 20s.

**Comment by sophokles73** [comment]
Let me try again on my machine if I see the same result. I am pretty sure that this worked when I tested it ...

**Comment by kaniyan** [comment]
> 
> 
> @calohmn @kaniyan When did you run the `is_healthy.sh` command? Did it take that long on first invovation only and once the grid was up the command took only a second or less? Or does it take that long for you on each invocation?

In my machine, it did take long 30 seconds after the grid was up.
